src/js/helpers/reviewHelper.js

```diff
@@ -172,7 +172,7 @@ const getFileStates = (status) => {
 		else {
 			let count = 0;
 			item.error_data.forEach((error) => {
-				count += parseInt(error.occurrences);
+				count += parseInt(error.occurrences, 10);
```

Comment:
I think we should keep it if the linting rules are checking for it. Being explicit never hurt so I'm definitely ok with leaving it in.